tests/ts_aggregation_simple.erl

```diff
@@ -77,7 +84,9 @@ verify_aggregation(ClusterType) ->
        <<"COUNT(temperature)">>,
        <<"COUNT(precipitation)">>
       ],
-      [{Count, Count, Count}]},
+      [{count_non_nulls(?PRESSURE_COL_INDEX, Data),
```

Comment:
It is because the `COUNT` is called with one column so `NULL` is not counted in the total e.g. `COUNT(precipitation)`.  The counting on the test side now checks for `NULL` in the data written as well.